docs/application/native/guides/app-management/widget-viewer.md

```diff
@@ -0,0 +1,140 @@
+# EFL Widget Viewer
+
+
+The Widget Viewer API provides functionality that displays and manages Tizen Native widget applications.
```

Comment:
As this page is under Tizen Native Guide category, how about removing "Tizen Native" from this sentence?

The Widget Viewer API provides functionality that displays and manages widget applications.

Reply:
It is fine.